src/viam/sdk/resource/resource_server_base.cpp

```diff
@@ -1,5 +1,10 @@
 #include <viam/sdk/resource/resource_server_base.hpp>
 
 namespace viam {
-namespace sdk {}  // namespace sdk
+namespace sdk {
+const std::shared_ptr<SubtypeService>& ResourceServerBase::get_sub_svc() const {
```

Comment:
[nit] It seems like we have decided on `attribute()`, not `get_attribute()` semantics. I think this should be `sub_svc()` to be consistent.

Reply:
@stuqdog Is there a reason you didn't do this swap?

Reply:
nope! slipped the mind when I was updating, apologies :) we can add it as a flyby in a future feature though!